PiecewiseTrajectory trajectory
---
---
uint8 FOLLOWING = 0
uint8 DEADLOCKED = 1
uint8 CRASHED = 2
uint8 current_state # one of the above
float64 current_time_on_trajectory